guides/flex-layout.md

```diff
@@ -47,42 +47,52 @@ The `iron-flex-layout` component provides simple ways to use [CSS flexible box l
 *   Layout classes. The layout class stylesheet provides a simple set of class-based flexbox rules. Layout classes
     let you specify layout properties directly in markup.
 
-*   Custom CSS mixins.  The mixin stylesheet includes custom CSS mixins that can be applied 
-    inside a CSS rule using the `@apply` function. 
+*   Custom CSS mixins.  The mixin stylesheet includes custom CSS mixins that can be applied
+    inside a CSS rule using the `@apply` function.
 
-Using the classes or CSS mixins is largely a matter of preference. The following sections discuss 
+Using the classes or CSS mixins is largely a matter of preference. The following sections discuss
 how to use the each of the stylesheets.
 
-<aside><b>Note:</b> Before using either of these stylesheets, it's helpful to be familiar with the basics 
-of flexbox layout. Chris Coyier's [A Complete Guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) is a 
+<aside><b>Note:</b> Before using either of these stylesheets, it's helpful to be familiar with the basics
+of flexbox layout. Chris Coyier's [A Complete Guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) is a
 good primer.</aside>
 
 ### Using layout classes
 
-To use layout classes import the `classes/iron-flex-layout` file.
+To use layout classes import the `iron-flex-layout-classes` file. You
+must do this in any element that uses any of the `iron-flex-layout` styles.
 
-    <link rel="import" href="bower_components/iron-flex-layout/classes/iron-flex-layout.html">
+    <link rel="import" href="bower_components/iron-flex-layout/iron-flex-layout-classes.html">
+
+Then include the module(s) that you need:
+
+    <style is="custom-style" include="iron-flex iron-flex-alignment">
 
 Then simply apply the classes to any element.
 
     <div class="layout horizontal wrap">
 
-Many of the layout rules involve combinations of multiple classes (such as `layout horizontal wrap` above).
+Many of the layout rules involve combinations of multiple classes (such as `layout horizontal wrap` above),
+and will need a combination of modules.
 The order in which the classes are specified doesn't matter, so `layout horizontal` and `horizontal layout`
 are equivalent.
 
-_Currently_, the layout class stylesheet uses the `/deep/` combinator 
-and therefore, works across all local DOM boundaries. 
+There are 5 [modules](https://github.com/PolymerElements/iron-flex-layout/blob/master/iron-flex-layout-classes.html)
+available: `iron-flex` (for basic flex layouts), `iron-flex-reverse` (for reverse flexbox layouts),
```

Comment:
Since we have more that a couple of items here, recommend breaking them out to a bulleted list like so:
```
-   `iron-flex`. Basic flex layouts.
-   `iron-flex-reverse`. Reverse flexbox layouts.
-   `iron-flex-alignment`.  Main axis, cross axis and self alignment.
-   `iron-flex-factors`. All the available flex factors.
-   `iron-positioning`. Generic, non-flexbox positioning helpers.
```